distributed-tracing/node-js-app/beachshirt/tracing.js

```diff
@@ -0,0 +1,16 @@
+const jaegerTracer = require('jaeger-client').initTracer;
+
+module.exports.initTracer = (serviceName) => {
+  const config = {
+    serviceName: serviceName,
+    sampler: {
+      type: "const",
+      param: 1,
+    },
+    reporter: {
+       agentHost: "127.0.0.1",
+       agentPort: 6832,
+    },
+  };
+  return jaegerTracer(config);
+};
```

Comment:
L5: just **serviceName,** is sufficient.

const name = "John";
const obj = { name };
const obj2 = { name: name };

Both obj and obj2 have the same key and values, but just less typing.